libs/klv/src/main/java/org/codice/alliance/libs/klv/Utilities.java

```diff
@@ -40,6 +40,12 @@ public static boolean isAlphanumericString(Serializable serializable) {
     return serializable instanceof String && StringUtils.isAlphanumeric((String) serializable);
   }
 
+  // Used to see if certain security attributes are explicitly "None" so they are not put on the
+  // metacard.
+  public static boolean isNotStringNone(Serializable serializable) {
+    return serializable instanceof String && (!((String) serializable).trim().equals("None"));
```

Comment:
✏️ I think we should do `StringUtils.equalsIgnoreCase()` instead of `.equals()` just to be safe (and your unit test will have to change accordingly).

Reply:
fixed